pkg/secrets/secrets.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2023 Adevinta
+*/
+
+package secrets
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	vterrors "github.com/adevinta/vulcan-tracker/pkg/errors"
+	"github.com/labstack/echo/v4"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	awsconfig "github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/secretsmanager"
+)
+
+const region = "eu-west-1"
+
+type (
+	// Secrets manage the credentials of ticket trackers servers.
+	Secrets interface {
+		GetServerCredentials(serverID string) (Credentials, error)
+	}
+
+	// AWSSecrets represents the specific AWS secret manager.
+	AWSSecrets struct {
+		Logger echo.Logger
+	}
+
+	// Credentials store the credentials of a secret.
+	Credentials struct {
+		User     string `json:"user"`
+		Password string `json:"password"`
+	}
+)
+
+// GetServerCredentials return the user and password inside a Credentials type
+// from AWS secret manager for a specific server.
+func (s *AWSSecrets) GetServerCredentials(serverID string) (Credentials, error) {
```

Comment:
Not necessary for this PR but I think we should refactor this component in the future in the following way:
* Create a function that functions that constructs and returns an instance of the AWSSecrets struct that stores a ``*secretsmanager.Client`` in a field of type `SecretsManagerAPI` defined in the package `github.com/aws/aws-sdk-go-v2/service/secretsmanager/secretsmanageriface` . That way it will be easy add tests for the component and also we won't need to create an aws session each time we call the method `GetServerCredentials`.  [Here](https://github.com/adevinta/vulcan-scan-engine/blob/master/pkg/notify/snsnotifier.go#L39) you can find an example of that kind of implementation.

Reply:
Fixed in commit ce9cece908f3f3ed5744069c61cfd9540f824107